towerlib/towerlib.py

```diff
@@ -850,6 +850,79 @@ def create_organization_inventory(self,
             raise InvalidOrganization(organization)
         return organization_.create_inventory(name, description, variables)
 
+    @property
+    def inventories_scripts(self):
+        """The inventories configured in tower.
+
+        Returns:
+            list of Inventory: The inventories configured in tower.`
+
+        """
+        return EntityManager(self,
+                             entity_name='inventory_scripts',
+                             entity_object='InventoryScript',
+                             primary_match_field='name')
+
+    def create_organization_inventory_script(self,
+                                name,
```

Comment:
Please align these properly.

Reply:
Since the method is called create_organization_inventory_script I would expect to be asked for the organization as the first argument